docs/source/api/apollo-server.md

```diff
@@ -133,9 +133,13 @@ The `registerServer` method is from `apollo-server-express`. Middleware registra
 
     Specify a custom path. It defaults to `/graphql` if no path is specified.
 
-  * `cors`: <`Object`>
+  * `cors`: <`Object`> (express, hapi)
 
-    Pass the cors options.
+    Pass the integration specific cors options.
```

Comment:
"integration-specific" with hyphen.  Maybe document that this is equivalent to the arguments to the cors or hapi-cors packages, with links?